channels/candidate-4.3.yaml

```diff
@@ -1,5 +1,6 @@
 name: candidate-4.3
 versions:
+- 4.2.16
```

Comment:
in fast-4.3 we did 4.2.16+amd64

Reply:
Fixed with 0a03965, which also links #41.